### PR TITLE
XFail R.swift and RxDataSources

### PR DESCRIPTION
**Pull request by xedin**
https://bugs.swift.org/browse/SR-11135
https://bugs.swift.org/browse/SR-11136

### CODE REVIEW

**Comment by xedin** [comment]
@swift-ci please test

File: projects.json [hunk 1]
```diff
@@ -1868,7 +1868,21 @@
       {
         "action": "BuildSwiftPackage",
         "configuration": "release",
-        "tags": "sourcekit"
+        "tags": "sourcekit",
+        "xfail": {
+          "compatibility": {
+            "4.2": {
+              "branch": {
+                "master": "https://bugs.swift.org/browse/SR-11135"
```
Comment (clackary) [inline projects.json]:
```suggestion
                "master": "https://bugs.swift.org/browse/SR-11135",
                "swift-5.1-branch": "https://bugs.swift.org/browse/SR-11135"
```

File: projects.json [hunk 2]
```diff
@@ -1868,7 +1868,21 @@
...
+              }
+            },
+            "5.0": {
+              "branch": {
+                "master": "https://bugs.swift.org/browse/SR-11135"
```
Comment (clackary) [inline projects.json]:
```suggestion
                "master": "https://bugs.swift.org/browse/SR-11135",
                "swift-5.1-branch": "https://bugs.swift.org/browse/SR-11135"
```

**Comment by clackary** [comment]
@swift-ci test

**Comment by clackary** [comment]
@swift-ci test

**Comment by theblixguy** [comment]
Can we merge this now? Thanks!

**Pull request merged**